src/pages/iou/request/step/IOURequestStepWaypoint.tsx

```diff
@@ -160,6 +160,7 @@ function IOURequestStepWaypoint({
             onEntryTransitionEnd={() => textInput.current?.focus()}
             shouldEnableMaxHeight
             testID={IOURequestStepWaypoint.displayName}
+            style={styles.overflowHidden}
```

Comment:
Same here, I might be wrong, but it look like this change isn't needed anymore

Reply:
@eVoloshchak good catch, I just removed it 😄